util/logger.go

```diff
@@ -125,3 +127,21 @@ func (log *RigLogger) Verbose(format string, a ...interface{}) {
 func (log *RigLogger) Note(format string, a ...interface{}) {
 	log.Channel.Info.Println(fmt.Sprintf(format, a...))
 }
+
+// PrivilegeEscallationPrompt interrupts a running spinner to ensure clear
+// prompting to the user for sudo password entry. It is up to the caller to know
+// that privilege is needed. This prompt is only displayed on the first privilege
+// escallation of a given rig process.
+func (log *RigLogger) PrivilegeEscallationPrompt() {
+	defer func() { log.Privileged = true }()
+
+	if log.Privileged {
+		return
+	}
+
+	// This newline ensures the last status before escallation is preserved
+	// on-screen.
+	fmt.Println()
+	log.Spin("Evaluating administrative action...")
```

Comment:
Just noting that things go by so quickly on my machine I never see the "Evaluating administrative action..." message.

Reply:
I wouldn't expect it to. I need the log.Spin() to start the spinner and the log.Warning() for the real message... let me test just using the same message for both, since that would avoid coder confusion.